docs/technical-documentation/rest-signposting.md

```diff
@@ -0,0 +1,119 @@
+(XML, JSON, JSONLD, etc...).
+
+As a general precaution, link headers for Drupal entities are not exposed to users that do not have the permissions to view
+the entity linked in the header.  So making GET and HEAD requests as anonymous will yield a different set of headers than
+what an authenticated user would see.  For example, anonymous users don't have the `view media` permission, so they will not
+see the link headers for media associated with a node.
+
+## Link Headers Provided by Islandora
+
+### Alternate Representations
+Other representations generated by different serializers available through Drupal's REST API are exposed as link headers
+with `rel="alternate"` and `type` equal to the mimetype that will be received when dereferencing the link.  For example,
+if an entity in Drupal has a JSONLD representation, then the link header returned in a GET or HEAD response would look like
+
+`Link: <http://example.org/node/1?_format=jsonld>; rel="alternate"; type="application/ld+json"`
+
+### Referenced Entities
+
+Entity reference fields are exposed as link headers with `rel="related"` and a title equal to the entity reference field's display label.
+For example, if `http://example.org/node/1` has an entity reference field name "Associated Content" that references 
+`http://example.org/node/2`, then the link header returned in a GET or HEAD response would look like
+
+`Link: <http://example.org/node/2>; rel="related"; title="Associated Content"`
+
+### Referenced Taxonomy Terms
+
+Entity reference fields for taxonomy terms get special handling.  The taxonomy terms used to tag content are exposed as link headers 
+with `rel="tag"` and a title equal to the taxonomy term's display label.  If the term has an external uri in a controlled vocabulary, 
+then that uri is provided.  Otherwise, the local Drupal uri is provided.  For example, if a piece of content is tagged with 
+`taxonomy/term/1`, which has a display label of "Example Term", then the link header returned in a GET or HEAD response would look like 
+
+`Link: <http://example.org/taxonomy/term/1>; rel="tag"; title="Example Term"`
+
+If instead the term were to have the `field_external_uri` field with a value of `http://exampletwo.org/vocab#term` then the link
+header would look like 
+
+`Link: <http://exampletwo.org/vocab#term>; rel="tag"; title="Example Term"`.   
+
+### Associated Media
+
+Media entities belonging to nodes are exposed as link headers with `rel="related"` and a title equal to the display label of
+their `field_media_use` taxonomy term.  For example, if a Media is tagged as `Original File` indicating
+that it is the initial file uploaded, the link header returned in a GET or HEAD response for a node would look like
+
+`Link: <http://example.org/media/1>; rel="related"; title="Original File"`.
+
+### Source Files
+
+Files that are the source for Media entities are exposed as Link headers in the GET and HEAD responses with `rel="describes"`.
```

Comment:
What constitutes a source file? Is it a file wrapped in a media that is tagged as "Original File"?

Reply:
It's _the_ file for a Media, regardless of what usage tag has been slapped on it.  That's a good clarification to make.

Reply:
... and you made it by using "File" instead of "Media". My bad. Please ignore, I'll resolve this conversation.